Implementation/application/controllers/SingleAuction.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+    class SingleAuction extends CI_Controller{
+
+        public function __construct(){
+            parent::__construct();
+
+            $this->load->model('Auction');
+            $this->load->model('UserBids');
+
+            $id = $this->input->get("id");
+            $auction = $this->Auction->getAuctionById($id)[0];
```

Comment:
Jel moglo ovo da se uradi da imamo polje auction: 
`
private $auction;
`
i onda u konstruktoru odradis:
 `id = $this->input->get("id");
$auction = $this->Auction->getAuctionById($id)[0];`
, a u ostalim funkcijama ne moras da radis isto to, nego uvek imas objekat `$auction`?